example/MOCKSERVER.md

```diff
@@ -0,0 +1,98 @@
+# Mock Server
+
+### How to run Mockserver
+1. [Install Mockserver](https://github.com/namshi/mockserver#installation).
+```
+npm install -g mockserver
```

Comment:
@saturninoabril I added the package dependency in case standalone mockserver is not launched. check `example/detox/e2e/init.js` Let me know if you're still having issues. Thanks!

Reply:
I also add `pod install` in the doc